lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -239,6 +241,15 @@ export class FormFieldModel extends FormWidgetModel {
     }
 
     private getDefaultDateFormat(jsonField: any): string {
+        if(jsonField.fields) {
+            Object.keys(jsonField.fields).forEach((el) => {
+                if(jsonField.fields[el]) {
+                    jsonField.fields[el].forEach((element) => {
+                        element.dateDisplayFormat = element.dateDisplayFormat? DateFnsUtils.convertMomentToDateFnsFormat(element.dateDisplayFormat): element.dateDisplayFormat;
```

Comment:
Please add the space before '?'.
What if element.dateDisplayFormat is null?

Reply:
fixed the linting. 
This already includes null and '' checks.